master/internal/rm/multirm/multirm_intg_test.go

```diff
@@ -719,21 +720,33 @@ func TestGetRMName(t *testing.T) {
 		rpName         string
 		err            error
 		expectedRMName string
+		rmConflicts    []string
```

Comment:
I have a different take on this.

Let's just sort rmNames here https://github.com/determined-ai/determined/blob/e8b0165413fbe1142eea29de56fd860e698f8248/master/internal/rm/multirm/multirm.go#L22

I also like think generally having one canonical error message for the same issue is better than being able to return a bunch

Reply:
agree that sorting on rmNames is the best option -- pushed a new commit with that